### ISSUE TITLE
Failure to schedule PVC from cloudsnapshot volume

### ISSUE THREAD

**Issue by cehoffman** [libopenstorage/stork#62]
**Is this a BUG REPORT or FEATURE REQUEST?**: BUG

**What happened**:

Tried creating a PVC and PV from a cloud snapshot restore.

**What you expected to happen**:

A scheduled pod.

**How to reproduce it (as minimally and precisely as possible)**:

Restore a volume from cloudsnap. Create a PV with name equal to volume name and volume id field matching volume id. Create PVC referencing the PV. Create pod using PVC and stork scheduler. See in events.

> Post http://stork-service.kube-system.svc:8099/filter: EOF

**Anything else we need to know?**:

With stork 1.0.2

~~~
2018/04/15 05:59:14 http: panic serving 10.2.9.135:46030: runtime error: invalid memory address or nil pointer dereference
goroutine 131556 [running]:
net/http.(*conn).serve.func1(0xc4200feaa0)
	/home/travis/.gimme/versions/go1.8.3.linux.amd64/src/net/http/server.go:1721 +0xd0
panic(0x11e98a0, 0x1bf4280)
	/home/travis/.gimme/versions/go1.8.3.linux.amd64/src/runtime/panic.go:489 +0x2cf
github.com/libopenstorage/stork/drivers/volume/portworx.(*portworx).InspectVolume(0xc420184aa0, 0xc4202e1ef5, 0x5, 0x1d, 0xc420238401, 0x0)
	/home/travis/gopath/src/github.com/libopenstorage/stork/drivers/volume/portworx/portworx.go:100 +0x22b
github.com/libopenstorage/stork/drivers/volume/portworx.(*portworx).GetPodVolumes(0xc420184aa0, 0xc42024c000, 0x3, 0xc4205bcbe0, 0x1, 0x1, 0x0)
	/home/travis/gopath/src/github.com/libopenstorage/stork/drivers/volume/portworx/portworx.go:194 +0x3f7
github.com/libopenstorage/stork/pkg/extender.(*Extender).processFilterRequest(0xc420310120, 0x1baea60, 0xc420106000, 0xc420601700)
	/home/travis/gopath/src/github.com/libopenstorage/stork/pkg/extender/extender.go:122 +0x3c1
github.com/libopenstorage/stork/pkg/extender.(*Extender).serveHTTP(0xc420310120, 0x1baea60, 0xc420106000, 0xc420601700)
	/home/travis/gopath/src/github.com/libopenstorage/stork/pkg/extender/extender.go:79 +0x8a
github.com/libopenstorage/stork/pkg/extender.(*Extender).(github.com/libopenstorage/stork/pkg/extender.serveHTTP)-fm(0x1baea60, 0xc420106000, 0xc420601700)
	/home/travis/gopath/src/github.com/libopenstorage/stork/pkg/extender/extender.go:48 +0x48
net/http.HandlerFunc.ServeHTTP(0xc420232370, 0x1baea60, 0xc420106000, 0xc420601700)
	/home/travis/.gimme/versions/go1.8.3.linux.amd64/src/net/http/server.go:1942 +0x44
net/http.(*ServeMux).ServeHTTP(0x1c07cc0, 0x1baea60, 0xc420106000, 0xc420601700)
	/home/travis/.gimme/versions/go1.8.3.linux.amd64/src/net/http/server.go:2238 +0x130
net/http.serverHandler.ServeHTTP(0xc4200b5b80, 0x1baea60, 0xc420106000, 0xc420601700)
	/home/travis/.gimme/versions/go1.8.3.linux.amd64/src/net/http/server.go:2568 +0x92
net/http.(*conn).serve(0xc4200feaa0, 0x1baf9e0, 0xc42040ce80)
	/home/travis/.gimme/versions/go1.8.3.linux.amd64/src/net/http/server.go:1825 +0x612
created by net/http.(*Server).Serve
	/home/travis/.gimme/versions/go1.8.3.linux.amd64/src/net/http/server.go:2668 +0x2ce
~~~

**Environment**:
- Kubernetes version (use `kubectl version`): 1.8.9
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by disrani-px** [comment]
@cehoffman This has been fixed in v1.0.3 with Issue #45

**Comment by cehoffman** [comment]
@disrani-px Thanks for the new release.
